src/booleans.js

```diff
@@ -1,61 +1,65 @@
-function negate(a) {
-  // your code here
+const negate = (a) => {
+  return !(a);
 };
 
-function both(a, b) {
-  // your code here
+const both = (a, b) => {
+  return (a && b);
 };
 
-function either(a, b) {
-  // your code here
+const either = (a, b) => {
+  return (a || b);
 };
 
-function none(a, b) {
-  // your code here
+const none = (a, b) => {
+  return !(a || b)
 };
 
-function one(a, b) {
-  // your code here
+const one = (a, b) => {
+  return ((a && !b) || (!a && b));
 };
 
-function truthiness(a) {
-  // your code here
+const truthiness = (a) => {
+  if (a) {
+    return true
+  } else {
+    return false
+  }
 };
 
-function isEqual(a, b) {
-  // your code here
+const isEqual = (a, b) => {
+  return (a === b);
 };
 
-function isGreaterThan(a, b) {
-  // your code here
+const isGreaterThan = (a, b) => {
+  return (a > b);
 };
 
-function isLessThanOrEqualTo(a, b) {
-  // your code here
+const  isLessThanOrEqualTo = (a, b) => {
+  return (a <= b);
 };
 
-function isOdd(a) {
-  // your code here
+const isOdd = (a) => {
+  return (a % 2 != 0);
 };
 
-function isEven(a) {
-  // your code here
-};
+const isEven = (a) => {
+  return (a % 2 == 0);
+}
 
-function isSquare(a) {
-  // your code here
+const isSquare = (a) => {
+  return Math.sqrt(a) % 1 === 0;
 };
 
-function startsWith(char, string) {
-  // your code here
+const startsWith = (char, string) => {
+  return string.startsWith(char);
 };
 
-function containsVowels(string) {
-  // your code here
+const containsVowels = (string) => {
+  return string.match('a|e|i|o|u|A|E|I|O|U/i') ? true : false;
 };
 
-function isLowerCase(string) {
-  // your code here
+const isLowerCase = (string) => {
+  return string == string.toLowerCase() && string != string.toUpperCase();
```

Comment:
This comparison doesnt make a lot of sense, you are comparing if the string is all lower case with the first statement, and if the string is not all uppercase with the second one. This means this will wrongly pass if a test is given for an all capital string.
Instead you could just test if the string is all lower case as follows:
```suggestion
  return string === string.toLowerCase();
```